app/waf/src/controllers/adc.controller.ts

```diff
@@ -243,6 +245,25 @@ export class AdcController extends BaseController {
     }
   }
 
+  async installPartition(adc: Adc): Promise<void> {
+    // Install partition after installing the AS3 agent.
+    let tenantName = adc.getAS3Name();
+    let mgmt = adc.management!;
+    let paritionObj = new AS3PartitionRequest(adc);
+    try {
+      await this.serialize(adc, {status: AdcState.PARTITIONING});
+      await this.asgMgr.deploy(mgmt.ipAddress, mgmt.tcpPort, paritionObj);
+      await this.serialize(adc, {status: AdcState.ACTIVE});
+    } catch (err) {
+      this.logger.debug(`Creating partition ${tenantName} Error.`);
```

Comment:
you might want 2 use logger.error instead.

Reply:
corrected.